doc/sphinx/index.rst

```diff
@@ -1,11 +1,29 @@
 Azure Storage for PyTorch Documentation
 =======================================
 
-Azure Storage integrations for PyTorch.
+Azure Storage for PyTorch (``azstoragetorch``) is a library that provides
+seamless, performance-optimized integrations between  `Azure Storage`_ and `PyTorch`_.
+Use this library to easily access and store data in Azure Storage while using PyTorch. The
+library currently supports:
+
+* :ref:`Saving and loading PyTorch models (i.e., checkpointing) to and from Azure Blob Storage <checkpoint-guide>`
+
+Visit the :ref:`Getting Started <getting-started>` page for more information on how to start using
+Azure Storage for PyTorch.
+
+User Guide
+----------
+.. toctree::
+
+   user-guide
+
 
 API Reference
 -------------
 .. toctree::
    :maxdepth: 2
 
    api
+
+.. _Azure Storage: https://learn.microsoft.com/en-us/azure/storage/common/storage-introduction
```

Comment:
For all learn links, we should manually remove the `en-us` from the URL so that it will direct to user's default locale.

Reply:
Good catch. I copied these URLs post redirection it seems. I'll update before merging.